### ISSUE TITLE
Sentry Ansible errors (being ignored?)

### ISSUE THREAD

**Issue by cdrage** [CentOS/container-pipeline-service#481]
I'm getting these two errors but they're being ignored:

```
TASK [repo_tracking : Initialize for creating DB tables.] ****************************************************************************************************************************************************************************************$
fatal: [192.168.10.10]: FAILED! => {"changed": true, "cmd": "/opt/cccp-service/manage.py makemigrations --noinput", "delta": "0:00:00.456799", "end": "2018-02-20 21:03:31.066824", "msg": "non-zero return code", "rc": 1, "start": "2018-02-20 2$
:03:30.610025", "stderr": "Traceback (most recent call last):\n  File \"/opt/cccp-service/manage.py\", line 13, in <module>\n    execute_from_command_line(sys.argv)\n  File \"/usr/lib64/python2.7/site-packages/django/core/management/__init__.$
y\", line 363, in execute_from_command_line\n    utility.execute()\n  File \"/usr/lib64/python2.7/site-packages/django/core/management/__init__.py\", line 337, in execute\n    django.setup()\n  File \"/usr/lib64/python2.7/site-packages/django$
__init__.py\", line 22, in setup\n    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)\n  File \"/usr/lib64/python2.7/site-packages/django/utils/log.py\", line 75, in configure_logging\n    logging_config_func(logging_settings)\n  
File \"/usr/lib64/python2.7/logging/config.py\", line 803, in dictConfig\n    dictConfigClass(config).configure()\n  File \"/usr/lib64/python2.7/logging/config.py\", line 585, in configure\n    '%r: %s' % (name, e))\nValueError: Unable to con$
igure handler 'sentry': Cannot resolve 'raven.handlers.logging.SentryHandler': No module named raven", "stderr_lines": ["Traceback (most recent call last):", "  File \"/opt/cccp-service/manage.py\", line 13, in <module>", "    execute_from_co$
mand_line(sys.argv)", "  File \"/usr/lib64/python2.7/site-packages/django/core/management/__init__.py\", line 363, in execute_from_command_line", "    utility.execute()", "  File \"/usr/lib64/python2.7/site-packages/django/core/management/__i$
it__.py\", line 337, in execute", "    django.setup()", "  File \"/usr/lib64/python2.7/site-packages/django/__init__.py\", line 22, in setup", "    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)", "  File \"/usr/lib64/python2.7/$
ite-packages/django/utils/log.py\", line 75, in configure_logging", "    logging_config_func(logging_settings)", "  File \"/usr/lib64/python2.7/logging/config.py\", line 803, in dictConfig", "    dictConfigClass(config).configure()", "  File $
"/usr/lib64/python2.7/logging/config.py\", line 585, in configure", "    '%r: %s' % (name, e))", "ValueError: Unable to configure handler 'sentry': Cannot resolve 'raven.handlers.logging.SentryHandler': No module named raven"], "stdout": "", $
stdout_lines": []}
...ignoring

TASK [repo_tracking : Create DB tables] **********************************************************************************************************************************************************************************************************$
fatal: [192.168.10.10]: FAILED! => {"changed": true, "cmd": "/opt/cccp-service/manage.py migrate --noinput", "delta": "0:00:00.395216", "end": "2018-02-20 21:03:32.538817", "msg": "non-zero return code", "rc": 1, "start": "2018-02-20 21:03:32$
143601", "stderr": "Traceback (most recent call last):\n  File \"/opt/cccp-service/manage.py\", line 13, in <module>\n    execute_from_command_line(sys.argv)\n  File \"/usr/lib64/python2.7/site-packages/django/core/management/__init__.py\", l$
ne 363, in execute_from_command_line\n    utility.execute()\n  File \"/usr/lib64/python2.7/site-packages/django/core/management/__init__.py\", line 337, in execute\n    django.setup()\n  File \"/usr/lib64/python2.7/site-packages/django/__init$
_.py\", line 22, in setup\n    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)\n  File \"/usr/lib64/python2.7/site-packages/django/utils/log.py\", line 75, in configure_logging\n    logging_config_func(logging_settings)\n  File \$
/usr/lib64/python2.7/logging/config.py\", line 803, in dictConfig\n    dictConfigClass(config).configure()\n  File \"/usr/lib64/python2.7/logging/config.py\", line 585, in configure\n    '%r: %s' % (name, e))\nValueError: Unable to configure $
andler 'sentry': Cannot resolve 'raven.handlers.logging.SentryHandler': No module named raven", "stderr_lines": ["Traceback (most recent call last):", "  File \"/opt/cccp-service/manage.py\", line 13, in <module>", "    execute_from_command_l$
ne(sys.argv)", "  File \"/usr/lib64/python2.7/site-packages/django/core/management/__init__.py\", line 363, in execute_from_command_line", "    utility.execute()", "  File \"/usr/lib64/python2.7/site-packages/django/core/management/__init__.py
\", line 337, in execute", "    django.setup()", "  File \"/usr/lib64/python2.7/site-packages/django/__init__.py\", line 22, in setup", "    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)", "  File \"/usr/lib64/python2.7/site-pac
kages/django/utils/log.py\", line 75, in configure_logging", "    logging_config_func(logging_settings)", "  File \"/usr/lib64/python2.7/logging/config.py\", line 803, in dictConfig", "    dictConfigClass(config).configure()", "  File \"/usr/l
ib64/python2.7/logging/config.py\", line 585, in configure", "    '%r: %s' % (name, e))", "ValueError: Unable to configure handler 'sentry': Cannot resolve 'raven.handlers.logging.SentryHandler': No module named raven"], "stdout": "", "stdout_
lines": []}
...ignoring
```

Are they important?

**Comment by cdrage** [comment]
Well, looks like they really were (and I don't know why they ended up being ignored) since they created the database tables for the code to communicate with.

Doing a `pip install sentry` seems to have fixed the problem. Adding this to the ansible provisioning script seemed to resolve it.. Incoming PR!

**Comment by mohammedzee1000** [comment]
@cdrage thanks !!

**Comment by cdrage** [comment]
Closed via #488
